### DISCUSSION TITLE
does not implement ModalComponent interface on fresh installation

### DISCUSSION THREAD

**Discussion by holiccoder** [wire-elements/modal#480] [Q&A]
tested with laravel 10 and laravel 11.
![02159](https://github.com/user-attachments/assets/030fdaab-c2f6-49af-9a78-42825666ab43)
[LivewireUI\Modal\Modal] does not implement [LivewireUI\Modal\Contracts\ModalComponent] interface.

**Comment by auto-nic** [comment]
I have the same... completely fresh install (everything else in my installation is working as intended).

Where do we start looking?
